Bio/PDB/PDBList.py

```diff
@@ -102,19 +102,22 @@ def __init__(self, server='ftp://ftp.wwpdb.org', pdb=os.getcwd(),
 
     @staticmethod
     def _print_default_format_warning(file_format):
-        """Temporary warning (similar to a deprecation warning) that files are being downloaded in mmCIF"""
+        """Print a warning to stdout.
```

Comment:
Should this end with '(PRIVATE)'  like in `Bio/Nexus/Nexus.py` so as to keep this consistent?

Reply:
Yes, especially since it is so short. Good spot.